analysis.json

```diff
@@ -10,12 +10,12 @@
           "affectedVersions": {
             "[8.0.0,8.17.0)": {
               "status": "TRUE_POSITIVE",
-              "comment": "The risk is true, update to 8.17.0",
+              "comment": "The jsoup dependency is a transitive dependency of the vaadin-server dependency. The affected versions which contain the jsoup vulnerability are [8.0.0, 8.17.0). The vulnerability is patched in jsoup 1.15.3. Please update to Vaadin 8.17.0, which already contains the patched jsoup dependency.",
```

Comment:
Using ranges in the maven format here could be confusing, I would use words like from/to/including

Reply:
Thank you. I updated the comment.